sklearn/naive_bayes.py

```diff
@@ -460,6 +463,16 @@ def _update_class_log_prior(self, class_prior=None):
         else:
             self.class_log_prior_ = np.zeros(n_classes) - np.log(n_classes)
 
+    def _check_alpha(self):
```

Comment:
Maybe you went through this before, but why isn't alpha just initially set to an appropriate value and then used as before, instead of changing the code a lot as below?